pkg/relayer/session/session.go

```diff
@@ -95,23 +98,24 @@ func (rs *relayerSessionsManager) EnsureSessionTree(sessionHeader *sessiontypes.
 // goListenToCommittedBlocks listens to committed blocks so that rs.sessionsToClaimPublisher
```

Comment:
Need to update the comment.

1. I think providing a comment saying "it returns a list of sessions that can be claimed given a single block..." would help
2. Should it be `mapBlcok**s**To...` or `mapBlockTo...`